monkey/monkey_island/cc/app.py

```diff
@@ -109,12 +113,31 @@ class CustomConfirmRegisterForm(ConfirmRegisterForm):
             "Email", default="dummy@dummy.com", validators=[validate_no_user_exists_already]
         )
 
+    from flask_login import user_loaded_from_request
+
+    @user_loaded_from_request.connect
+    def user_loaded_from_request(self, user=None):
+        g.login_via_request = True
```

Comment:
What does this mean? What is `g`?

Reply:
https://flask.palletsprojects.com/en/2.2.x/api/#flask.g
![image](https://user-images.githubusercontent.com/15820737/222716938-3de7d38d-0b0c-4f54-9b31-1591d521c954.png)
We are practically saying that we are going to login in from request and in that case we don't want to save a session.

Reply:
Ahh, so the `g` is flasks' global context